aldryn_forms/contrib/email_notifications/models.py

```diff
@@ -0,0 +1,187 @@
+# -*- coding: utf-8 -*-
+from email.utils import formataddr
+from functools import partial
+
+from django.conf import settings
+from django.core.exceptions import ValidationError
+from django.db import models
+from django.utils.translation import ugettext, ugettext_lazy as _
+
+from djangocms_text_ckeditor.fields import HTMLField
+
+from aldryn_forms.models import FormPlugin
+
+from emailit.api import construct_mail
+
+from .helpers import get_theme_template_name, render_text
+
+
+EMAIL_THEMES = getattr(
+    settings,
+    "ALDRYN_FORMS_EMAIL_THEMES",
+    [('default', _('default'))]
```

Comment:
If default is always available, wouldn't it be better to append the themes from settings, rather than override?